### ISSUE TITLE
six not longer in django

### ISSUE THREAD

**Issue by ZixiaoShi** [nesdis/djongo#341]
I believe djongo imports six from django.utils? I think it is removed from django 3.

**Comment by 4rch18** [comment]
Yes, it's mentioned in the [Django 3.0 Release Notes](https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis)

> django.utils.six - Remove usage of this vendored library or switch to six.

### Error faced
The use of django.utils.six in operations.py (/venv/lib/python3.7/site-packages/djongo⁩/) `from django.utils import six,timezone` gives the error `ImportError: cannot import name 'six' from 'django.utils' (/venv/lib/python3.7/site-packages/django/utils/__init__.py)`

Versions used:
- Django==3.0
- djongo==1.2.38

**Comment by 4rch18** [comment]
Pull request for fixing the issue: https://github.com/nesdis/djongo/pull/353
